pkg/gui/commit_message_panel.go

```diff
@@ -29,8 +30,19 @@ func (gui *Gui) RenderCommitLength() {
 	}
 
 	gui.Views.CommitMessage.Subtitle = getBufferLength(gui.Views.CommitMessage)
+	gui.checkCommitLengthWarning()
 }
 
 func getBufferLength(view *gocui.View) string {
 	return " " + strconv.Itoa(strings.Count(view.TextArea.GetContent(), "")-1) + " "
 }
+
+func (gui *Gui) checkCommitLengthWarning() {
+	commitLength, _ := strconv.Atoi(strings.Trim(gui.Views.CommitMessage.Subtitle, " "))
```

Comment:
we shouldn't depend on the view's subtitle to access the length of the content. If we wanted to just access the integer I'd change `getBufferLength` to return an int i.e. directly return `strings.Count(view.TextArea.GetContent(), "")-1` and then in  `RenderCommitLength` I'd go:
```go
commitLength := getBufferLength(gui.Views.CommitMessage)
gui.Views.CommitMessage.Subtitle = fmt.Sprintf(" %d ", commitLength)
```

With that said: when people talk about limiting the length of a commit message's lines: they're talking about a line-by-line basis, so looking at the entirety of the commit message won't suffice: we need to instead check if any line exceeds the threshold.

Reply:
I didn't even consider that, I have a `zshrc` config that points if I've passed 72 chars only in the commit title. But yeah it makes a lot of sense to extend this to multiple lines.